wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/ConfigurationMessageCell/ConversationMessageSectionController.swift

```diff
@@ -112,6 +112,7 @@ final class ConversationMessageSectionController: NSObject, ZMMessageObserver {
 
     private let userSession: UserSession
     private let privateDefaults: PrivateUserDefaults<CollapseKey>
+    private var isChatBubbleSimpleEnabled: Bool = false
```

Comment:
issue: I think this can be like the following

```suggestion
    private let isChatBubbleSimpleEnabled: Bool
```

Reply:
Done